measures/emissions_reporting/measure.rb

```diff
@@ -509,9 +518,9 @@ def run(runner, user_arguments)
     puts natural_gas_query
     natural_gas_values = sql_file.execAndReturnVectorOfDouble(natural_gas_query).get
     if natural_gas_values.empty?
-      runner.registerWarning('Unable to get hourly timeseries facility natural gas use from the model, the model may not use gas.  Cannot calculate emissions.')
+      runner.registerWarning('Unable to get hourly timeseries facility natural gas use from the model, the model may not use gas. Cannot calculate emissions.')
     else
-      annual_natural_gas_emissions_co2e_kg = natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_co2e_kg_per_kbtu }.sum
+      annual_natural_gas_emissions_co2e_kg += natural_gas_values.map { |v| v * j_to_kbtu * natural_gas_emissions_factor_co2e_kg_per_kbtu }.sum
```

Comment:
This change is fine but unnecessary; there is no place later in the code that modifies this value.